ocs_ci/ocs/ui/views.py

```
@@ -1856,17 +1860,18 @@
 acm_configuration_4_22 = {
-    "single-perspective": (
-        "//*[@id='only-one-perspective']",
+    "perspective-switcher-toggle": (
+        "//*[@data-test-id='perspective-switcher-toggle']",
         By.XPATH,
     ),
-    "all-clusters_dropdown": (
-        "//button[@data-test-id='perspective-switcher-toggle']",
+    "all-clusters_dropdown_item": (
+        "//button[@role='option'][normalize-space()='Fleet management'] | "
+        "//button[@role='option'][normalize-space()='All Clusters']",
```

Ruff (0.16.2)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)